<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>779</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* { font-family: arial; }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pbConfig">
          <property name="styleSheet">
           <string notr="true">QPushButton {margin: 0; padding: 6px; border: 1px solid #aaa; background: #ccc;}
QPushButton:hover { background: #eee; }
QPushButton:pressed { background: #ddd; }
</string>
          </property>
          <property name="text">
           <string>Config</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbAddNew">
          <property name="styleSheet">
           <string notr="true">QPushButton {margin: 0; padding: 6px; border: 1px solid #aaa; background: #ccc;}
QPushButton:hover { background: #eee; }
QPushButton:pressed { background: #ddd; }
</string>
          </property>
          <property name="text">
           <string>Add New...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="twEnvSets">
          <property name="styleSheet">
           <string notr="true">QTreeView::item {
min-height: 24px;
}</string>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>1</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>USD-23.11</string>
           </property>
           <item>
            <property name="text">
             <string>Debug</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Release</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Release-Python</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Custom1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Custom2</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>USD-23.08</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>USD-23.02</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>USD-22.11</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QStackedWidget" name="swPage">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Preference</string>
           </property>
           <layout class="QFormLayout" name="formLayout_3">
            <property name="labelAlignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Install Root Directory</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="leInstallRootDirectory"/>
              </item>
              <item>
               <widget class="QToolButton" name="tbInstallRootDirectory">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Download Temporary Directory</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="leDownloadTempDirectory"/>
              </item>
              <item>
               <widget class="QToolButton" name="tbDownloadTempDirectory">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Tools</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <widget class="QTreeWidget" name="twToolchains">
              <column>
               <property name="text">
                <string>Tool Display Name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Commandline</string>
               </property>
              </column>
              <item>
               <property name="text">
                <string>Roah</string>
               </property>
               <item>
                <property name="text">
                 <string>Roah Offline Rendering</string>
                </property>
                <property name="text">
                 <string>python -m start.roah_offline_renderer</string>
                </property>
               </item>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0,1">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background: #444; border-radius: 6px; padding: 4px; color: white; font-family: Arial;</string>
           </property>
           <property name="text">
            <string>USD-23.11 : Debug</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Tools</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QListWidget" name="listWidget_3">
               <property name="iconSize">
                <size>
                 <width>60</width>
                 <height>60</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>Console</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>UsdView...</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>UsdView</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QListWidget" name="listWidget_4"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_4">
            <attribute name="title">
             <string>Usd Config</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <layout class="QFormLayout" name="formLayout_2">
               <property name="labelAlignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>USD Library</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="comboBox">
                 <item>
                  <property name="text">
                   <string>Custom</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>OpenUSD-23.11 : Debug</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>OpenUSD-23.11 : Release</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>OpenUSD-23.11 : Release with Python3.9</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>bin</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLineEdit" name="lineEdit_3"/>
                 </item>
                 <item>
                  <widget class="QToolButton" name="toolButton_3">
                   <property name="text">
                    <string>...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>lib</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLineEdit" name="lineEdit_4"/>
                 </item>
                 <item>
                  <widget class="QToolButton" name="toolButton_4">
                   <property name="text">
                    <string>...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>python</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLineEdit" name="lineEdit_5"/>
                 </item>
                 <item>
                  <widget class="QToolButton" name="toolButton_5">
                   <property name="text">
                    <string>...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_12">
               <property name="text">
                <string>Download Prebuild Package and Install</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>490</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_5">
            <attribute name="title">
             <string>Plugins Config</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <widget class="QWidget" name="formWidget_2" native="true">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="2" column="1">
                 <widget class="QListWidget" name="listWidget_5"/>
                </item>
                <item row="1" column="1">
                 <widget class="QListWidget" name="listWidget_2"/>
                </item>
                <item row="0" column="1">
                 <widget class="QListWidget" name="listWidget"/>
                </item>
                <item row="0" column="2">
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="QPushButton" name="pushButton_3">
                    <property name="text">
                     <string>Add</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_5">
                    <property name="text">
                     <string>Add...</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_4">
                    <property name="text">
                     <string>Remove</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="verticalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>External Plugins</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <widget class="QPushButton" name="pushButton_6">
                    <property name="text">
                     <string>Add</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_7">
                    <property name="text">
                     <string>Add...</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_8">
                    <property name="text">
                     <string>Remove</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="verticalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>External PythonPath</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="2" column="2">
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <item>
                   <widget class="QPushButton" name="pushButton_9">
                    <property name="text">
                     <string>Add</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_10">
                    <property name="text">
                     <string>Add...</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_11">
                    <property name="text">
                     <string>Remove</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="verticalSpacer_4">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>External UsdView Plugins</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_6">
            <attribute name="title">
             <string>Extra Environments</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_14">
             <item>
              <widget class="QPlainTextEdit" name="plainTextEdit"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_3">
            <attribute name="title">
             <string>Develop</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Environments Preview</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="plainTextEdit_2"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>977</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Menu">
    <property name="title">
     <string>&amp;Menu</string>
    </property>
    <addaction name="actionOpenGithub"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu_Menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenGithub">
   <property name="text">
    <string>&amp;Open Github</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
